qiskit_machine_learning/kernels/quantum_kernel.py

```diff
@@ -47,13 +52,15 @@ class QuantumKernel:
     algorithms such as support vector classification, spectral clustering or ridge regression.
     """
 
+    @deprecate_arguments("0.5.1", {"user_parameters": "training_parameters"})
```

Comment:
This is the version at which it was deprecated. This would be 0.5.0 as per the [VERSION.txt](https://github.com/Qiskit/qiskit-machine-learning/blob/main/qiskit_machine_learning/VERSION.txt) (0.5.1 would be a bug fix release and deprecation is not something that would be done on a bug fix release). Same for the other deprecated aspects in this PR.

Reply:
Hi, this is a "starter PR" (joined the S3 team); @calebj15 suggested it, there's an issue ( https://github.com/qiskit-community/prototype-quantum-kernel-training/issues/33 ) and I'm working thru it.  I'll correct these two problems posthaste.

Reply:
Oops, also, forgot about the release-note, will fix. Also, I don't know what release user_parameters will get removed at?  Do you have a suggestion?

Reply:
I'm writing a release-note, and it isn't clear to me what the file should be named.  That is, sure, it should be named "rename-user_parameters-to-training_parameters-7d8cf1e056b967a7.yaml", but should I put it in the releasenotes/notes directory, or in a new "0.5" subdirectory ?

Reply:
Also, "reno report" stops at version "0.3", does not proceed to format releasenotes for version "0.4".  Not sure why.

Reply:
> Also, I don't know what release user_parameters will get removed at? Do you have a suggestion?

The version in the deprecation message is the version at which point the deprecation was made (commenced) so in the messages it needs to be 0.5.0 which will be the next version released and the point when this code/PR is included. We do not state the point at which it will be removed, beyond saying at least 3 months or 2 releases - see https://qiskit.org/documentation/contributing_to_qiskit.html#deprecation-policy for more info - where bullet point 3a talks about the 2 release aspect.

> I'm writing a release-note, and it isn't clear to me what the file should be named. 

You just need to do `reno new some_name_for_your_fix` from the root of the project - see here for more info https://github.com/Qiskit/qiskit-machine-learning/blob/main/CONTRIBUTING.md#release-notes And that will create a yaml filem, with the commit reference in the name too, in the `releasenotes` folder. You should edit it and commit it there. The notes are only moved later into version sub-folder just prior to release in order to keep them organized.

If you want to see the release notes in formatted form then the easiest way is to do `make html` in the project root to build the documentation for ML. This will create a `_build` folder under `docs`, and in there will be an `html` folder where you can see release_notes.html. You can click on it directly there is an index.html that is the root of the docs, and has a link to release notes in that.

Reply:
> Hi, this is a "starter PR" (joined the S3 team)

Hi, I heard you had just started. I imagined you had discussed it with someone but I was lacking any context and its helpful to have that. For instance in this case, as you did in the comment, the issue in the other repo could have been included as context.

Reply:
OK, I've fixed the various problems; should I delete this PR and resubmit with a single commit (to minimize clutter) ?  Or leave as-is?